components/http_client.rst

```diff
@@ -908,6 +908,110 @@ This allows using them where native PHP streams are needed::
     // later on if you need to, you can access the response from the stream
     $response = stream_get_meta_data($streamResource)['wrapper_data']->getResponse();
 
+Extensibility
+-------------
+
+In order to extend the behavior of a base HTTP client, decoration is the way to go::
+
+    class MyExtendedHttpClient implements HttpClientInterface
+    {
+        private $decoratedClient;
+
+        public function __construct(HttpClientInterface decoratedClient = null)
```

Comment:
you missed a dollar `public function __construct(HttpClientInterface $decoratedClient = null)`

Reply:
fixed thanks